src/App.tsx

```diff
@@ -60,11 +60,11 @@ function App() {
           <Route path="/two-fa-setup" element={<Setup2FA />} />
           <Route path="/verify-code" element={<VerifyCode />} />
           <Route path="/seller-home" element={<Dashboard />} />
-          <Route
-            path="/seller-home/notifications"
-            element={<SellerNotification />}
-          />
-          <Route path="/seller-home/chat" element={<Chatting />} />
+          {/* <Route
+          path="/seller-home/notifications"
```

Comment:
Also here

Reply:
I have taken this as a base branch for my feature of viewing a cart, am suggesting if i can take care of this it would be great. Thank you.